### PR TITLE
chore: displaying social icons with valid social URL

### PR DESCRIPTION
**Pull request by s2sharpit**
## Related Issue

Closes: #1982 

## Description
displaying social icons with valid social URL

## Changes
- implemented logic to show the icons only when the string is not empty
- for GitHub, adding URL based on the collected GitHub username as some users are not adding GitHub account URL. 

## Screenshots

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| image | image |

### CODE REVIEW

**Comment by priyankarpal** [comment]
@s2sharpit we need to change the `json` format 
- delete extra social accounts code
- also, for github delete `"gitHub": "https://github.com/Sahilll15",`

**Comment by s2sharpit** [comment]
> @s2sharpit we need to change the `json` format
> 
> * delete extra social accounts code
> * also, for github delete `"gitHub": "https://github.com/Sahilll15",`

Sure @priyankarpal

**Comment by s2sharpit** [comment]
> > @s2sharpit we need to change the `json` format
> > 
> > * delete extra social accounts code
> > * also, for github delete `"gitHub": "https://github.com/Sahilll15",`
> 
> Sure @priyankarpal

Done @priyankarpal

File: DB/projects.json [hunk 1]
```diff
@@ -219,11 +212,8 @@
   {
     "github_username": "adity2911",
     "Social_media": {
-      "gitHub": "https://github.com/adity2911",
       "LinkedIn": "https://www.linkedin.com/in/aditya-tulsiyan-585b50111/",
-      "Instagram": "YOUR_INSTAGRAM_ACCOUNT_LINK",
-      "YouTube": "YOUR_YOUTUBE_ACCOUNT_LINK",
-      "Twitter": "YOUR_TWITTER_ACCOUNT_LINK"
+      "Twitter": ""
```
Comment (priyankarpal) [inline DB/projects.json:216]: what about blank social link? can we avoid it?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 1 above)
Comment (s2sharpit) [reply DB/projects.json:216]: Yaa, we can avoid them. Let me do that.

Comment (s2sharpit) [reply DB/projects.json]: Hey @priyankarpal, I have removed the blank social link. Kindly review it now.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by priyankarpal** [comment]
@s2sharpit the changes have been made, so we need to update the documentation as well. You can create a separate issue for the documentation.

**Comment by s2sharpit** [comment]
> @s2sharpit the changes have been made, so we need to update the documentation as well. You can create a separate issue for the documentation.

Yaa Sure

**Pull request merged**